src/pages/household/output/EarningsVariation/BaselineOnlyChart.jsx

```diff
@@ -110,8 +115,29 @@ export default function BaselineOnlyChart(props) {
         style={{
           width: "100%",
         }}
+        onHover={(data) => {
+          const netIncome = data.points[0].y?.toLocaleString("en-US", 
+            { style:"currency",
+              currency:"USD",
```

Comment:
Does this show pounds for the UK view?

Reply:
Since it was hard coded as `USD` this did not properly show pounds for the UK view. Following how the policy charts handled displaying currency, I made the change of appending the `metadata.currency` character to the string returned from the `.toLocaleString` method.